CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+# v1.4.0 (Aug 2020)
+  * Objective-C/Swift interop:
+    * Reworked exception handling ([GH-4191](https://github.com/JetBrains/kotlin-native/pull/4191))
+    * Enabled support for Objective-C generics by default ([GH-3778](https://github.com/JetBrains/kotlin-native/pull/3778))
+    * Support for Kotlin’s suspending functions ([GH-3915](https://github.com/JetBrains/kotlin-native/pull/3915))
+    * .dSYMs for release binaries ([GH-4085](https://github.com/JetBrains/kotlin-native/pull/4085))
+  * Added native-specific frontend checkers (implemented in the main Kotlin repository: [GH-3293](https://github.com/JetBrains/kotlin/pull/3293), [GH-3091](https://github.com/JetBrains/kotlin/pull/3091), [GH-3172](https://github.com/JetBrains/kotlin/pull/3172))
+  * Reworked cinterop under the hood
```

Comment:
More details are required.
For example, it is worth mentioning that it improves compilation time with cinterop involved.

Reply:
Out of curiosity, is the improvement in OSX only or across the board?

Reply:
It affects all supported targets.